concepts/chapel.scroll

```diff
@@ -6,15 +6,29 @@ appeared 2004
 creators David Callahan and Hans Zima and Brad Chamberlain and John Plevyak
 tags pl arrayLang
 website https://chapel-lang.org/
-blog https://chapel-lang.org/whatsnew.html
+spec https://chapel-lang.org/docs/language/spec/index.html
+blog https://chapel-lang.org/blog/index.html
 releaseNotes https://chapel-lang.org/releaseNotes.html
-latestVersion 2.0.0
-lab Cray
+download https://chapel-lang.org/download.html
+latestVersion 2.2.0
+conferences https://chapel-lang.org/ChapelCon.html
+lab Cray Inc.
 fileExtensions chpl
-writtenIn chapel cpp c restructuredtext assembly-language python bash cmake bourne-shell make html m4 xml toml markdown perl json tex c-shell ocaml yaml pascal javascript css svg vim-script csharp dockerfile diff lisp llvmir protobuf ini fortran-90 objective-c fortran-77 dtd lex bazel ruby jupyter-notebook cuda php typescript rust csv yacc d haskell nix cython go java awk xslt julia sed
```

Comment:
note that this is pulled directly from the repo. do you think we should override with yours?

Reply:
@breck7 : Fast turnaround time, thanks!

Good question.  I figured that must be pulled from the repo in some way, but even then couldn't quite make sense of some of the choices, like Julia, OCaml, Rust, etc.  These are things we mention in our repo, but don't rely upon that I'm aware of.  Our best guess was that we had files checked into our test system that caused false positives here, but even then…

In putting my list together, I focused on the things we rely upon the utmost, which felt like what I'd be interested in when browsing the site.  That said, I didn't notice until now that that's an auto-computed field, and if it makes things simpler for you maintenance-wise to keep auto-computing it from the repo, I'd completely support that.

Reply:
It clones the main repo and runs cloc. By hand is good too though.